pages/packingDetailed.js

```diff
@@ -0,0 +1,333 @@
+        status: this.state.status,
+      }),
+      headers: { "Content-Type": "application/json", Authorization: token },
+    }).then(() => {
+      this.setState({ success: "Changed order status successfully!" });
+      setTimeout(() => this.setState({ success: null }), 1000);
+    });
+  };
+
+  changeStatusOfItem(barcode) {
+    this.state.items[barcode].isPacked = !this.state.items[barcode].isPacked;
+    this.setState({ items: this.state.items });
+    fetch("/api/orders/SetOrderItemStatus", {
+      method: "POST",
+      body: JSON.stringify({
+        orderId: this.state.orderId,
+        itemId: barcode,
+        isPacked: this.state.items[barcode].isPacked,
+      }),
+      headers: { "Content-Type": "application/json", Authorization: token },
+    }).then(() => {});
+  }
+
+  displayUnpackedItemRow(barcode, value) {
+    var itemName = this.state.itemMap[barcode].itemName;
+    return (
+      <tr className="h-10" key={barcode}>
+        <td className="">
+          <div className="float-left">
+            <button
+              className="font-bold text-xl"
+              onClick={() => this.changeStatusOfItem(barcode)}
+            >
+              {itemName}
+            </button>
+          </div>
+        </td>
+        <td>
+          <h2 className="float-left text-xl">{value.quantity}</h2>
+        </td>
+      </tr>
+    );
+  }
+
+  displayPackedItemRow(barcode, value) {
+    var itemName = this.state.itemMap[barcode].itemName;
+    return (
+      <tr className="h-10" key={barcode}>
+        <td className="">
+          <div className="float-left">
+            <button
+              className="font-bold text-xl line-through"
+              onClick={() => this.changeStatusOfItem(barcode)}
+            >
+              {itemName}
+            </button>
+          </div>
+        </td>
+        <td>
+          <h2 className="line-through float-left text-xl">{value.quantity}</h2>
+        </td>
+      </tr>
+    );
+  }
+
+  displayOrderStatus() {
+    if (this.state.status == ORDER_STATUS_COMPLETE) {
+      return (
+        <h1 className="inline font-bold text-l pl-2">
+          Status:
+          <span>&#9989;</span>
+        </h1>
+      );
+    } else if (this.state.status == ORDER_STATUS_OPEN) {
+      return (
+        <h1 className="inline font-bold text-l pl-2">
+          Status:
+          <span>&#9898;</span>
+        </h1>
+      );
+    } else if (this.state.status == ORDER_STATUS_PROCESSING) {
+      return (
+        <h1 className="inline font-bold text-l pl-2">
+          Status:
+          <span>&#9881;</span>
+        </h1>
+      );
+    }
+  }
+
+  displayChangeOrderStatus() {
+    var textForButton = "";
+    if (this.state.status == ORDER_STATUS_COMPLETE) {
+      textForButton = "Mark as incomplete";
+      return (
+        <button
+          className="float-right bg-blue-300 hover:bg-blue-300 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline"
```

Comment:
Could instead try using `btn btn-pantry-blue` or `btn btn-outline` from [here](https://github.com/CalNourish/pantry-webapp/blob/main/styles/index.css). Matches pretty well with the rest of the theme/colors too.